<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="info_stack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>350</width>
     <height>710</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="chat_widget">
    <property name="styleSheet">
     <string notr="true">#chat_widget{background-color:white}</string>
    </property>
    <widget class="QListWidget" name="chat_show">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>350</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#d0a8a9;border-width:0;border-style:outset</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QWidget" name="chat_input_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>350</width>
       <height>161</height>
      </rect>
     </property>
     <widget class="QPushButton" name="chat_send">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
    color: rgb(255,255,255);
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(18,111,237), stop : 1 rgb(45,147,250));
}

QPushButton:hover {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(3,92,227), stop:1 rgb(21,133,244));
}

QPushButton:pressed {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(0,77,201), stop : 1 rgb(4,117,231));
}

QPushButton:disabled {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(196,196,196), stop : 1 rgb(207,207,207));
}</string>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="chat_input">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>350</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;border-width:0;border-style:outset</string>
      </property>
     </widget>
     <zorder>chat_input</zorder>
     <zorder>chat_send</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="member_widget">
    <widget class="QListWidget" name="member_show">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>350</width>
       <height>710</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#d0a8a9;border-width:0;border-style:outset</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="chat_info">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none
QPushButton:pressed
{
    background-color: rgb(255, 255, 0);
    border:2px solid #5F92B2; 
    border-radius:5px; 
    color:black; 
}</string>
   </property>
   <property name="text">
    <string>聊天</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="member_info">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>0</y>
     <width>175</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none
QPushButton:pressed
{
    background-color: rgb(255, 255, 0);
    border:2px solid #5F92B2; 
    border-radius:5px; 
    color:black; 
}</string>
   </property>
   <property name="text">
    <string>成员</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="background_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#b08384</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>background_frame</zorder>
  <zorder>info_stack</zorder>
  <zorder>chat_info</zorder>
  <zorder>member_info</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
